pkg/client/k8s/k8s_test.go

```diff
@@ -35,15 +35,18 @@ func TestGetConfigMap(t *testing.T) {
 	}
 
 	for _, test := range tests {
-		kc := &K8sClient{
-			ConfigMap: test.cm,
-		}
+		t.Run("TestGetConfigMap", func(t *testing.T) {
```

Comment:
Change the type of tests from []struct{..} to map[string]struct{}
The string thing will be the name of the test.

Reply:
Okay. I will do that.

Reply:
Actually I was not sure about the test names to put right now. Can you help me with that.